Panic when send iOS notification form CLI.

log:

```
ERRO[2017/05/11 - 23:26:46] Cert Error:open gs.p12: no such file or directory
DEBU[2017/05/11 - 23:26:46] Start push notification for iOS
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x13655ac]

goroutine 1 [running]:
github.com/appleboy/gorush/vendor/github.com/sideshow/apns2.(*Client).PushWithContext(0x0, 0x0, 0x0, 0xc42018ea10, 0x0, 0x0, 0x0)
	/Users/mtk10671/git/go/src/github.com/appleboy/gorush/vendor/github.com/sideshow/apns2/client.go:117 +0xdc
github.com/appleboy/gorush/vendor/github.com/sideshow/apns2.(*Client).Push(0x0, 0xc42018ea10, 0x1, 0x1, 0x7fff5fbfe285)
	/Users/mtk10671/git/go/src/github.com/appleboy/gorush/vendor/github.com/sideshow/apns2/client.go:99 +0x47
github.com/appleboy/gorush/gorush.PushToIOS(0xc42017dcd0, 0x1, 0x1, 0x1, 0x7fff5fbfe285, 0xc, 0x0, 0x0, 0x0, 0x0, ...)
	/Users/mtk10671/git/go/src/github.com/appleboy/gorush/gorush/notification.go:413 +0x20a
main.main()
	/Users/mtk10671/git/go/src/github.com/appleboy/gorush/main.go:245 +0xcfa
```

--- a/main.go
+++ b/main.go
@@ -210,7 +210,10 @@ func main() {
 			gorush.LogError.Fatal(err)
 		}
 
-		gorush.InitAppStatus()
+		if err := gorush.InitAppStatus(); err != nil {
+			return
+		}
+
 		gorush.PushToAndroid(req)
 
 		return
@@ -240,8 +243,13 @@ func main() {
 			gorush.LogError.Fatal(err)
 		}
 
-		gorush.InitAppStatus()
-		gorush.InitAPNSClient()
+		if err := gorush.InitAppStatus(); err != nil {
+			return
+		}
+
+		if err := gorush.InitAPNSClient(); err != nil {
+			return
+		}
 		gorush.PushToIOS(req)
 
 		return
@@ -261,8 +269,17 @@ func main() {
 		gorush.LogError.Fatal(err)
 	}
 
-	gorush.InitAppStatus()
-	gorush.InitAPNSClient()
+	if err = gorush.InitAppStatus(); err != nil {
+		return
+	}
+
+	if err = gorush.InitAPNSClient(); err != nil {
+		return
+	}
+
 	gorush.InitWorkers(gorush.PushConf.Core.WorkerNum, gorush.PushConf.Core.QueueNum)
-	gorush.RunHTTPServer()
+
+	if err = gorush.RunHTTPServer(); err != nil {
+		gorush.LogError.Fatal(err)
+	}
 }

